python/src/FejerAlgorithm_doc.i.in

```diff
@@ -0,0 +1,122 @@
+%feature("docstring") OT::FejerAlgorithm
+"Fejer Integration algorithm
+
+Available constructors:
+    FejerAlgorithm(*dimension=1, method=FEJERTYPE1*)
+
+    FejerAlgorithm(*discretization, method=FEJERTYPE1*)
+
+Parameters
+----------
+dimension : int, :math:`dimension>0`
+    The dimension of the functions to integrate. The default discretization is *FejerAlgorithm-DefaultMarginalIntegrationPointsNumber* in each dimension, see :class:`~openturns.ResourceMap`.
+discretization : sequence of int
+    The number of nodes in each dimension. The sequence must be non-empty and must contain only positive values.
+
+Notes
+-----
+The FejerAlgorithm algorithm enables to approximate the definite integral :
```

Comment:
in english there is no space before the ":"

Reply:
ok